governance/polls/Vault Compensation Working Group Payment - January 11, 2021.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Vault Compensation Working Group Payment - January 11, 2021
+summary: Signal your support or opposition to paying 12,700 Dai from the surplus buffer to the vault compensation working group for past labor.
+discussion_link: https://forum.makerdao.com/t/signal-request-vault-compensation-working-group-payment/5562
+vote_type: Plurality Voting
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Vault Compensation Working Group Payment - January 11, 2021
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Maker Governance Community. The community can vote in this poll to express support or opposition to transferring 12,700 Dai from the surplus buffer to members of the vault compensation working group for their past work organizing the vault compensation proposals. Funds will be allocated to individual members as follows:
```

Comment:
So I've started moving poll-specific detail to the review section. The only info under the top-level heading should be the meta-detail of the poll itself (ie when it is, how it got up on the portal, type of poll, etc.)

MIP9 Community Greenlight Poll - WFIL - December 14, 2020.md for an example.

Reply:
Awesome, thank you!

Reply:
updated in commit 63b048f0a5f8a741c627b2896faae94125c222b6